sklearn/compose/_column_transformer.py

```diff
@@ -597,6 +608,50 @@ def _get_transformer_list(estimators):
     return transformer_list
 
 
+def select_types(dtypes):
+    """Generate a column selector callable (type-based) to be passed to
```

Comment:
Pep257: a short summary should be alone on the first line

Reply:
Got it.